pkg/app/component_test.go

```diff
@@ -498,10 +519,15 @@ type bar struct {
 	appInstalled bool
 	appRezized   bool
 	updated      bool
+	title        string
```

Comment:
Here we want to test if OnInit gets called. Since we used a bool with the other test to check whether the event was called, we should keep the same pattern for consistency.

```go
initialized bool
```

Reply:
I can't set different values for bool. I wanted to test that it does not run multiple times. But as it seems the test doesn't actually really "dismount". I didn't care too much about it because I checked this in my real project.